fm_agent/configs/MPS3-tap-network.conf

```diff
@@ -0,0 +1,21 @@
+## Turn off terminal1 terminal2 and termimal5, keep terminal0 open
+mps3_board.telnetterminal1.quiet=1
+mps3_board.telnetterminal2.quiet=1
+mps3_board.telnetterminal5.quiet=1
+
+## Control the output format of the telnet terminal
+mps3_board.telnetterminal0.mode=raw
+## Suppress the telnet/xterm to be launched, so that model agent can talk to the port  
+mps3_board.telnetterminal0.start_telnet=0
+
+
+## Turn the rate limite off, make FVP runs as fast as possiable
+mps3_board.visualisation.rate_limit-enable=0
+
+
+## Turn on user Networking
```

Comment:
Could you please check the comment?

Do we need to create `ARMfmuser`  as a pre-requisite?

Reply:
Comment synced across both TAP config files, And yes it is expected that the user has created the `ARMfmuser` interface.

Reply:
Would be nice if can you add that as a comment and refer to some documentation, if possible.

Reply:
BTW, not creating the `ARMfmuser` is not a catastrophic failure. The FVP would still run (and execute the target binary) but there may be issues at runtime.